src/keys.rs

```diff
@@ -37,12 +35,15 @@ use crate::{
         PreparedNonIdentityBase, PreparedNonZeroScalar, PrfExpand,
     },
     zip32::{
-        self, ChildIndex, ExtendedSpendingKey, ZIP32_ORCHARD_PERSONALIZATION,
+        self, ExtendedSpendingKey, ZIP32_ORCHARD_PERSONALIZATION,
         ZIP32_ORCHARD_PERSONALIZATION_FOR_ISSUANCE,
     },
 };
 
-pub use zip32::{DiversifierIndex, Scope};
+// Disable rustfmt to preserve '::' which specifies the external 'zip32' crate,
+// avoiding confusion with a local module of the same name.
```

Comment:
just
`// preserve '::' which specifies the EXTERNAL 'zip32' crate`

Reply:
Fixed.